packages/lesswrong/components/posts/PostsPage/PostsPageSplashHeader.tsx

```diff
@@ -129,21 +129,22 @@ const styles = (theme: ThemeType) => ({
     paddingBottom: '7vh',
     transition: 'transform .75s ease-in-out',
     [theme.breakpoints.down('xs')]: {
-      paddingBottom: '1vh'
+      paddingBottom: '7vh'
```

Comment:
This seems unnecessary given L129 (would just delete it entirely if that's what you want it to be)